# translation of ru.po to Russian
# translation of firstboot.master.ru.po to
# translation of ru.po to
#
# Alexandr Kanevskiy <kad@blackcatlinux.com>, 2003.
# Leonid Kanter <leon@asplinux.ru>, 2004.
# Andrew Martynov <andrewm@inventa.ru>, 2004, 2005, 2006, 2008.
# Yuliya Poyarkova <ypoyarko@brisbane.redhat.com>, 2005, 2006.
# Yuliya Poyarkova <ypoyarko@redhat.com>, 2006.
# Yulia Poyarkova <ypoyarko@redhat.com>, 2006.
# Nikolay Sivov <bunglehead@gmail.com>, 2007.
# Yulia Poyarkova <yulia.poyarkova@redhat.com>, 2008, 2009.
# Copyright (C) 2003
msgid ""
msgstr ""
"Project-Id-Version: ru\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2009-03-12 16:50-0400\n"
"PO-Revision-Date: 2009-09-22 11:14+1000\n"
"Last-Translator: Yulia Poyarkova <yulia.poyarkova@redhat.com>\n"
"Language-Team: Russian <fedora-trans-ru@redhat.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: KBabel 1.11.4\n"

#: ../progs/firstboot:140
msgid "You must be root to run firstboot."
msgstr "Для запуска firstboot вам необходимо быть пользователем root."

#: ../progs/firstboot:163 ../progs/firstboot:164
msgid "Could not start any firstboot frontend."
msgstr "Невозможно запустить ни один из интерфейсов firstboot."

#: ../progs/firstboot:180 ../progs/firstboot:181
msgid "No firstboot modules were found."
msgstr "Не найдены модули firstboot."

#: ../progs/firstboot:190 ../progs/firstboot:191
msgid "Could not create any firstboot interface."
msgstr "Невозможно создать ни один из интерфейсов firstboot."

#: ../firstboot/exceptionWindow.py:50
#, python-format
msgid "An error has occurred in the %s module."
msgstr "Произошла ошибка в модуле %s."

#: ../firstboot/exceptionWindow.py:51
#, python-format
msgid ""
"Since there is a problem with the %s module,\n"
"firstboot will not load this module and will\n"
"attempt to run the remaining modules."
msgstr ""
"В связи с проблемой в модуле %s, \n"
"firstboot не загрузит этот модуль \n"
"и попробует запустить остальные модули."

#: ../firstboot/exceptionWindow.py:55
msgid "An error has occurred in firstboot."
msgstr "Произошла ошибка в firstboot."

#: ../firstboot/exceptionWindow.py:56
msgid "Since there is a problem, firstboot will exit."
msgstr "В связи с проблемой firstboot завершает свою работу."

#: ../firstboot/exceptionWindow.py:66
#, python-format
msgid ""
"A copy of the debug output has been saved to %s\n"
"Be sure to attach that file to the bug report.\n"
msgstr ""
"Копия отладочной информации сохранена в %s \n"
"Убедитесь, что вы прикрепили этот файл к отчету об ошибке.\n"

#: ../firstboot/exceptionWindow.py:71
msgid ""
"Please file a bug against 'firstboot' in the Red Hat\n"
"bug tracking system at http://www.redhat.com/bugzilla.\n"
msgstr ""
"Пожалуйста, создайте запрос для компонента firstboot в \n"
"системе обработки ошибок Red Hat по адресу http://www.redhat.com/bugzilla. \n"

#: ../firstboot/interface.py:75
msgid "Attempted to go back, but history is empty."
msgstr "При попытке вернуться назад обнаружено, что история возврата пуста."

#. If we were previously on the last page, we need to set the Next
#. button's label back to normal.
#: ../firstboot/interface.py:83 ../firstboot/interface.py:162
msgid "_Finish"
msgstr "_Готово"

#: ../firstboot/interface.py:183
msgid "The system must now reboot for some of your selections to take effect."
msgstr "Чтобы установки вступили в силу, необходим перезапуск системы."

#: ../firstboot/interface.py:243
msgid "_Back"
msgstr "_Назад"

#: ../firstboot/interface.py:250
msgid "_Forward"
msgstr "_Вперёд"

#: ../firstboot/interface.py:277
#, python-format
msgid "Module %s did not set up its UI, removing."
msgstr "Модуль %s не настроил свой пользовательский интерфейс и будет удален."

#: ../firstboot/interface.py:353 ../firstboot/interface.py:354
msgid "moveToPage must be given a module title or page number."
msgstr "moveToPage должен быть заданным заголовком модуля или номером страницы."

#: ../firstboot/interface.py:438
msgid "Unable to create the screenshot dir; skipping."
msgstr "Невозможно создать каталог снимков экрана, пропускаем."

#: ../firstboot/interface.py:478 ../firstboot/interface.py:479
#, python-format
msgid "No module exists with the title %s."
msgstr "Нет модуля с заголовком %s."

#: ../firstboot/loader.py:90
#, python-format
msgid "Skipping old module %s that has not been updated."
msgstr "Пропускаем старый модуль %s, который не был обновлен."

#: ../firstboot/loader.py:102
#, python-format
msgid "Module %s does not contain a class named moduleClass; skipping."
msgstr "Модуль %s не содержит класса moduleClass; пропускаем."

#: ../firstboot/loader.py:113
#, python-format
msgid "Module %s does not contain the required attribute %s; skipping."
msgstr "Модуль %s не содержит требуемого атрибута %s; пропускаем."

#: ../firstboot/module.py:187
#, python-format
msgid "Unable to load pixmap %s for module %s."
msgstr "Невозможно загрузить картинку %s для модуля %s."

#: ../firstboot/moduleset.py:93
#, python-format
msgid "Module %s did not set up its UI; removing."
msgstr "Модуль %s не настроил свой пользовательский интерфейс и будет удален."

#: ../modules/additional_cds.py:43 ../modules/additional_cds.py:44
msgid "Additional CDs"
msgstr "Дополнительные CD"

#: ../modules/additional_cds.py:55
msgid ""
"Please insert the disc labeled \"Red Hat Enterprise Linux Extras\" to allow "
"for installation of third-party plug-ins and applications.  You may also "
"insert the Documentation disc, or other Red Hat-provided discs to install "
"additional software at this time."
msgstr ""
"Вставьте диск с надписью \"Red Hat Enterprise Linux Extras\" для установки "
"приложений и дополнительных модулей (plug-in) третьих фирм.  Вы также можете "
"вставить диск с документацией или любой другой диск, поставляемый компанией "
"Red Hat, для установки дополнительного программного обеспечения на данном "
"этапе."

#: ../modules/additional_cds.py:61
msgid "Please insert any additional software install cds at this time."
msgstr "Вставьте диск с дополнительным программным обеспечением."

#: ../modules/additional_cds.py:65
msgid ""
"\n"
"\n"
"To enable runtime support of 32-bit applications on the Intel Itanium2 "
"architecture you must install the Intel Execution Layer package from the "
"Extras disc now."
msgstr ""
"\n"
"\n"
"Для выполнения 32-битных приложений на архитектуре Intel Itanium2, "
"необходимо дополнительно установить пакет Intel Execution Layer с диска с "
"дополнительным программным обеспечением (Extras)."

#: ../modules/additional_cds.py:73
msgid "Install..."
msgstr "Установка..."

#: ../modules/additional_cds.py:110
msgid ""
"A CD-ROM has not been detected.  Please insert a CD-ROM in the drive and "
"click \"OK\" to continue."
msgstr "CD-ROM найден.  Вставьте CD-ROM в привод и нажмите \"OK\" для продолжения."

#: ../modules/additional_cds.py:146
msgid "The autorun program cannot be found on the CD. Click \"OK\" to continue."
msgstr "Программа autorun не найдена на CD. Нажмите «OK» для продолжения."

#: ../modules/create_user.py:41 ../modules/create_user.py:42
msgid "Create User"
msgstr "Пользователь"

#: ../modules/create_user.py:83
msgid "You must create a user account for this system."
msgstr "Необходимо создать учётную запись пользователя для этой системы."

#: ../modules/create_user.py:94
msgid "You must enter and confirm a password for this user."
msgstr "Требуется ввести и подтвердить пароль пользователя."

#: ../modules/create_user.py:101
msgid "The passwords do not match.  Please enter the password again."
msgstr "Пароли не совпадают.  Введите пароль снова."

#: ../modules/create_user.py:113
#, python-format
msgid ""
"The username '%s' is a reserved system account.  Please specify another "
"username."
msgstr ""
"Пользователь %s -- зарезервированная системная учетная запись. Укажите "
"другое имя."

#: ../modules/create_user.py:132
#, python-format
msgid ""
"A home directory for user %s already exists. Would you like to continue, "
"making the new user the owner of this directory and all its contents?  Doing "
"so may take a while to reset permissions and any SELinux labels.  Would you "
"like to reuse this home directory?  If not, please choose a different "
"username."
msgstr ""
"Домашний каталог для пользователя %s уже существует. Вы действительно хотите "
"сделать нового пользователя владельцем этого каталога и его содержимого?  В "
"этом случае переназначение разрешений и меток SELinux может занять некоторое "
"время. Вы хотели бы повторно использовать этот домашний каталог? Если нет, "
"выберите другое имя пользователя."

#: ../modules/create_user.py:175
#, python-format
msgid "Fixing attributes on the home directory for %s.  This may take a few minutes."
msgstr ""
"Фиксируются атрибуты домашнего каталога для %s. Это может занять несколько "
"минут."

#: ../modules/create_user.py:195
#, python-format
msgid ""
"Problems were encountered fixing the attributes on some files in the home "
"directory for %s.  Please refer to %s for which files caused the errors."
msgstr ""
"Проблема при фиксации атрибутов некоторых файлов в домашнем каталоге для %s. "
"Обратитесь к %s для получения информации о том, какие файлы вызвали ошибки."

#: ../modules/create_user.py:213
msgid ""
"You must create a 'username' for regular (non-administrative) use of your "
"system.  To create a system 'username', please provide the information "
"requested below."
msgstr ""
"Требуется создать пользователя для повседневного (не административного) "
"использования системы. Для этого введите необходимые "
"данные."

#: ../modules/create_user.py:231
msgid "_Username:"
msgstr "_Имя пользователя:"

#: ../modules/create_user.py:238
msgid "Full Nam_e:"
msgstr "Полное им_я:"

#: ../modules/create_user.py:245
msgid "_Password:"
msgstr "_Пароль:"

#: ../modules/create_user.py:252
msgid "Confir_m Password:"
msgstr "Повто_р пароля:"

#: ../modules/create_user.py:261
msgid ""
"If you need to use network authentication, such as Kerberos or NIS, please "
"click the Use Network Login button."
msgstr ""
"Если требуется использовать проверку подлинности по сети, например "
"Kerberos или NIS, нажмите кнопку «Сетевая аутентификация»."

#: ../modules/create_user.py:270
msgid "Use Network _Login..."
msgstr "Сетевая аут_ентификация..."

#: ../modules/create_user.py:308
msgid "Please wait"
msgstr "Подождите"

#: ../modules/date.py:40 ../modules/date.py:41
msgid "Date and Time"
msgstr "Дата и время"

#: ../modules/date.py:62
msgid "Please set the date and time for the system."
msgstr "Пожалуйста, установите дату и время для системы."

#: ../modules/eula.py:35 ../modules/eula.py:36
msgid "License Information"
msgstr "Информация о лицензии"

#: ../modules/eula.py:53
msgid ""
"Thank you for installing Fedora.  Fedora is a compilation of software "
"packages, each under its own license.  The compilation is made available "
"under the GNU General Public License version 2.  There are no restrictions "
"on using, copying, or modifying this code.  However, there are restrictions "
"and obligations that apply to the redistribution of the code, either in its "
"original or a modified form.  Among other things, those restrictions/"
"obligations pertain to the licensing of the redistribution, trademark "
"rights, and export control.\n"
"\n"
"If you would like to understand what those restrictions are, please visit "
"http://fedoraproject.org/wiki/Legal/Licenses/LicenseAgreement."
msgstr ""
"Благодарим за установку Fedora. Fedora состоит из множества пакетов, каждый "
"из которых имеет свою собственную лицензию. Использование этих пакетов "
"возможно в рамках лицензии GNU General Public License версии 2. Никаких "
"ограничений по использованию, копированию или изменению данного кода нет. "
"Однако существуют ограничения и обязательства, позволяющие распространять "
"данное программное обеспечение только в оригинальном, неизмененном виде. Эти "
"ограничения имеют также отношение к лицензированию распространения, правам "
"на торговые марки и контролю использования.\n"
"\n"
"Если вы хотите узнать, в чем именно заключаются данные ограничения, посетите "
"http://fedoraproject.org/wiki/Legal/Licenses/LicenseAgreement."

#: ../modules/eula.py:64
msgid "Understood, please proceed."
msgstr "Понятно, можно продолжать."

#: ../modules/welcome.py:35 ../modules/welcome.py:36
msgid "Welcome"
msgstr "Добро пожаловать"

#: ../modules/welcome.py:45
msgid ""
"There are a few more steps to take before your system is ready to use.  The "
"Setup Agent will now guide you through some basic configuration.  Please "
"click the \"Forward\" button in the lower right corner to continue"
msgstr ""
"Осталось ещё несколько шагов, после которых ваша система будет готова к "
"работе.  В этом вам поможет Помощник по настройке. Для продолжения нажмите кнопку «Вперёд»"

